src/helpers/enrich-dataset.test.ts

```diff
@@ -0,0 +1,585 @@
+        const hubDataset = {
+            id: 'foo',
+            access: 'public',
+            slug: 'nissan::skyline-gtr',
+            size: 1,
+            type: 'CSV',
+            created: 1570747289000,
+            license: 'none',
+            supportedExtensions: ['WFSServer'],
+            url: 'https://sampleserver3.arcgisonline.com/arcgis/rest/services/Earthquakes/RecentEarthquakesRendered/MapServer/0',
+        };
+
+        const geojson = {
+            type: 'Feature',
+            properties: hubDataset
+        }
+
+        const { properties } = enrichDataset(geojson, hubsite);
+        expect(properties.accessUrlWFS).toBe('https://sampleserver3.arcgisonline.com/arcgis/services/Earthquakes/RecentEarthquakesRendered/MapServer/WFSServer?request=GetCapabilities&service=WFS');
+    });
+
+    it('should generate WMS distribution url if supported', () => {
+        const hubDataset = {
+            id: 'foo',
+            access: 'public',
+            slug: 'nissan::skyline-gtr',
+            size: 1,
+            type: 'CSV',
+            created: 1570747289000,
+            license: 'none',
+            supportedExtensions: ['WMSServer'],
+            url: 'https://sampleserver3.arcgisonline.com/arcgis/rest/services/Earthquakes/RecentEarthquakesRendered/MapServer/0',
+        };
+
+        const geojson = {
+            type: 'Feature',
+            properties: hubDataset
+        }
+
+        const { properties } = enrichDataset(geojson, hubsite);
+        expect(properties.accessUrlWMS).toBe('https://sampleserver3.arcgisonline.com/arcgis/services/Earthquakes/RecentEarthquakesRendered/MapServer/WMSServer?request=GetCapabilities&service=WMS');
+    });
+
+
+
+    it('should generate download link without query string if wkid is not present in spatialReference', () => {
+        const hubDataset = {
+            id: 'foo',
+            access: 'public',
+            slug: 'nissan::skyline-gtr',
+            size: 1,
+            type: 'CSV',
+            created: 1570747289000,
+            license: 'none',
+            url: 'https://services1.arcgis.com/jUJYIo9tSA7EHvfZ/arcgis/rest/services/DCAT_Test/FeatureServer/0',
+            layer: {
+                geometryType: 'esriGeometryPolygon',
+            },
+            server: {
+                spatialReference: {
+                    latestWkid: 3310,
+                },
+            },
+            identifier: 'CALFIRE::DCAT_Test',
+            supportedExtensions: ['WMSServer']
+        };
+
+        const geojson = {
+            type: 'Feature',
+            properties: hubDataset
+        }
+
+        const { properties } = enrichDataset(geojson, hubsite);
+        expect(properties.accessUrlWMS).toBe('https://services1.arcgis.com/jUJYIo9tSA7EHvfZ/arcgis/services/DCAT_Test/FeatureServer/WMSServer?request=GetCapabilities&service=WMS');
+    });
+    
+
+    it('should generate download link without query string if wkid is not present in spatialReference', () => {
```

Comment:
This looks like a duplicate test from line 363...

Reply:
Good catch! I have removed the duplicate test